src/Client.php

```diff
@@ -260,4 +260,9 @@ public function getListGroup($listId, $groupId)
 
         return json_decode($response->getBody());
     }
+
+    public function subscriberHash($email)
```

Comment:
I would rename this to `getSubscriberHash`.

Reply:
done